plugins/BEdita/Core/tests/TestCase/Model/Table/RolesTableTest.php

```diff
@@ -155,4 +156,23 @@ public function testFindMine()
 
         static::assertEquals($expected, $result);
     }
+
+    /**
+     * Test delete admin role
+     *
+     * @return void
+     *
+     * @covers ::beforeDelete
+     */
+    public function testDeleteAdminRole()
```

Comment:
It is generally better to rely on PHPUnit's expected exceptions:

```php
/**
 * ...
 * @expectedException \BEdita\Core\Exception\ImmutableResourceException
 * @expectedExceptionCode 403
 * @expectedExceptionMessage Could not delete "role" 1
 */
public function testDeleteAdminRole()
{
    $role = $this->Roles->get(RolesTable::ADMIN_ROLE);
    $this->Roles->delete($role);
}
```

This should do the work with slightly less code. 😃